modules/node.js

```diff
@@ -169,7 +169,7 @@ Node.prototype.shared = {
 		modules.peers.networkHeight({ normalized: false }, (err, networkHeight) => {
 			setImmediate(cb, null, {
 				broadhash: modules.system.getBroadhash(),
-				consensus: modules.peers.getLastConsensus() || null,
+				consensus: modules.peers.getLastConsensus(),
```

Comment:
Did you find the reason as to why `|| null` was there earlier?

Reply:
@SargeKhan No, actually consensus property is used here https://github.com/LiskHQ/lisk/blob/2312-unit-tests-for-node-getStatus/api/controllers/node.js#L101 in a weird way because if it's null we are converting it back to 0. So `|| null` was totally useless in my opinion. If you think it's better to fix it in a separate issue I can create one for that.

Reply:
I was just asking. I think `modules.peers.getLastConsensus()` might return `undefined`. And it makes sense to set the default values in the controller as opposed to modules.

Reply:
I could find only to places that we assign it's value. here: https://github.com/LiskHQ/lisk/blob/2312-unit-tests-for-node-getStatus/modules/peers.js#L70 and here https://github.com/LiskHQ/lisk/blob/2312-unit-tests-for-node-getStatus/modules/peers.js#L475. Seems like it always returns an integer?

Reply:
You are right! We don't need that check. I thought maybe on application start `getLastConsensus` may have undefined value.